index.js

```diff
@@ -159,6 +163,10 @@ app.on('browser-window-blur', (e, win) => {
 	unregisterAllShortcuts(win);
 });
 
+app.on('window-all-closed', () => {
```

Comment:
This is actually a little bit trivial since I am unregistering all shortcuts for a window when the window is closed, just want to have a double check here.